### ISSUE TITLE
setGPSMockLocation error: "java.lang.IllegalArgumentException: Service Intent must be explicit:”

### ISSUE THREAD

**Issue by buddygit** [bitbar/robotium-extensions#4]
Hi, 

I am trying to Mock location on my Android device using 'recorder-extensions-5.0.jar':

```
public void setUp() throws Exception {
    super.setUp();
    solo = new Solo(getInstrumentation());
    soloExt = new ExtSolo(getInstrumentation(), getActivity(), this.getClass().getCanonicalName(), getName());

    getActivity();
}

public void testLocationMock() {
    solo.waitForActivity("MainActivity");
    soloExt.setGPSMockLocation(37.5768, -122.0645, 0);
}
```

Test errors like this: "java.lang.IllegalArgumentException: Service Intent must be explicit:”

**Comment by buddygit** [comment]
Here is more info:
- Full error message: java.lang.IllegalArgumentException: Service Intent must be explicit: Intent { act=com.bitbar.testdroid.monitor.ScreenshotService }
- The error points to the line => soloExt = new ExtSolo(getInstrumentation(), getActivity(), this.getClass().getCanonicalName(), getName());
- Android SDK tools ver: 25.1.7
- The issue is reproducible when connected to: Marshmallow and Lollipop devices and emulators

**Comment by szprutamich** [comment]
Fixed with 5.3.2
